src/interpreter/patch_for_319.jl

```diff
@@ -232,17 +232,35 @@ end
             (nothrow | noub) || break
```

Comment:
Might want to also make this change in `is_terminator_or_phi` above: https://github.com/JuliaLang/julia/commit/2fc4e2d61155adbde10fcfa3dd8e465bdf780ed7.